fusion-plugin-apollo/src/plugin.js

```diff
@@ -34,6 +35,7 @@ export type DepsType = {
   schema: typeof GraphQLSchemaToken.optional,
   endpoint: typeof GraphQLEndpointToken.optional,
   getApolloClient: typeof ApolloClientToken,
+  getDataFromTree: typeof GetDataFromTreeToken,
```

Comment:
Looks like this just needs to be `typeof GetDataFromTreeToken.optional`. I'll go ahead and see if I can make that update inline.